db/migrate/20181031123357_fix_default_employment_details.rb

```diff
@@ -0,0 +1,9 @@
+class FixDefaultEmploymentDetails < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
This should be up - you should use either change OR up and down

Reply:
I suck at migration stuff! Fixed in 0de2eaa